packages/fhir-converter/src/templates/cda/Sections/Immunization.hbs

```diff
@@ -1,6 +1,6 @@
-{{#if (contains (toString (toJsonString msg)) '2.16.840.1.113883.10.20.22.2.2.1')}}
-    {{#with (getFirstCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.2.1')}}
-            {{#each (toArray 2_16_840_1_113883_10_20_22_2_2_1.entry) as |immEntry|}}
+{{#if (contains (toString (toJsonString msg)) '2.16.840.1.113883.10.20.22.2.2')}}
+    {{#with (getFirstCdaSectionsByTemplateId msg '2.16.840.1.113883.10.20.22.2.2')}}
+            {{#each (toArray this.[2_16_840_1_113883_10_20_22_2_2].entry) as |immEntry|}}
```

Comment:
@RamilGaripov now following - why was the OID changed? https://build.fhir.org/ig/HL7/cda-ccda-2.2/StructureDefinition-2.16.840.1.113883.10.20.22.2.2.1.html

Reply:
Included both OIDs to be used for immunizations